hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonTableInputFormat.java

```diff
@@ -565,7 +565,9 @@ public BlockMappingVO getBlockRowCount(Job job, CarbonTable table,
                   allSegments.getInvalidSegments(), toBeCleanedSegments));
           for (InputSplit extendedBlocklet : extendedBlocklets) {
             CarbonInputSplit blocklet = (CarbonInputSplit) extendedBlocklet;
-            blockletToRowCountMap.put(blocklet.getSegmentId() + "," + blocklet.getFilePath(),
+            String filePath = blocklet.getFilePath();
+            String blockName = filePath.substring(filePath.lastIndexOf("/") + 1);
```

Comment:
Whether have "\\" in  filePath? like for windows?

Reply:
added replace("\\", "/") to handle for windows